### PR TITLE
Announcement: Deprecate legacy IP addresses

### PR DESCRIPTION
**Pull request by AGS4NO**
Announcement for deprecation of legacy IP addresses.

### CODE REVIEW

File: content/articles/announcement-legacy-ip-addresses.md
Comment (weppos) [inline content/articles/announcement-legacy-ip-addresses.md:22]: We should not reference the Basic Reseller as most plans don't know that they are. Actually, it was used by anyone in the early days.

```suggestion
The following IP addresses were used by our first generation network infrastructure, and they are now officially deprecated in favor of the current [DNSimple name servers and IPs](/articles/dnsimple-nameservers/):
```

File: content/articles/announcement-legacy-ip-addresses.md
Comment (weppos) [inline content/articles/announcement-legacy-ip-addresses.md:37]: Why are we specifying plans?

File: content/articles/announcement-legacy-ip-addresses.md
Comment (weppos) [inline content/articles/announcement-legacy-ip-addresses.md:44]: Let's not mention vanity name servers. Anyone using that IP is impacted.

File: content/articles/announcement-legacy-ip-addresses.md
Comment (itsalyse) [reply content/articles/announcement-legacy-ip-addresses.md:22]: they were superseded in 2016

File: content/articles/announcement-legacy-ip-addresses.md
Comment (itsalyse) [inline content/articles/announcement-legacy-ip-addresses.md:37]:
```suggestion
Accounts with records or vanity name servers using the deprecated IP addresses will be impacted.
```

File: content/articles/announcement-legacy-ip-addresses.md
Comment (AGS4NO) [reply content/articles/announcement-legacy-ip-addresses.md]: Removed all mentions of vanity name servers

File: content/articles/announcement-legacy-ip-addresses.md
Comment (AGS4NO) [reply content/articles/announcement-legacy-ip-addresses.md]: Thanks @itsalyse I have updated the copy based on feedback.

File: content/articles/announcement-legacy-ip-addresses.md
Comment (itsalyse) [inline content/articles/announcement-legacy-ip-addresses.md:33]: is this blog url just a placeholder for the pending post?

**Review by itsalyse — APPROVED** [review]
Overall copy looks good, approved pending adding the date and updating the blog link

File: content/articles/announcement-legacy-ip-addresses.md
Comment (san983) [reply content/articles/announcement-legacy-ip-addresses.md:33]: I think so, I think it is intended to point to this one in draft => https://docs.google.com/document/d/1SF0nqkXflWMw8XNnLh1j1Gl5CmasbiprwvN7tXxSMjQ/edit / [Custom edge Part 2 blog post](https://github.com/dnsimple/dnsimple-business/issues/2033), right @AGS4NO?

**Review by san983 — APPROVED** [review]

File: content/articles/announcement-legacy-ip-addresses.md
Comment (AGS4NO) [reply content/articles/announcement-legacy-ip-addresses.md:33]: Yes, this will be the related blog post. I will update the date and link today to finalize the email

**Comment by AGS4NO** [comment]
Dates and blog post link for the cache edge expansion have been updated.

@san983 can you give this a final pass? Once we merge and verify, we can update the link in the mailer and run initial tests on https://github.com/dnsimple/dnsimple-app/pull/28338

**Review by san983 — APPROVED** [review]

**Pull request merged**
